tests/assets/db/bitwarden.json

```diff
@@ -43,7 +43,7 @@
         ],
         "username": "dpbx@fner.ws",
         "password": "ws5T@;_UB[Q|P!8'`~z%XC'JHFUbf#IX _E0}:HF,[{ei0hBg14",
-        "totp": null
+        "totp": "S3K3TPI5MYA2M67V"
```

Comment:
I thought it would be a good idea to have one item with totp example. I’ve also added it in the CSV sample.